### PR TITLE
Overview and extension manifest for VSIX marketplace

### PR DESCRIPTION
**Pull request by kythant**
Overview and extension manifest for VSIX marketplace.

For use in automated publishing of the VSIX.

### CODE REVIEW

File: dev/VSIX/extension.manifest.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vsix-publish",
+    "categories": [ "Build", "Coding", "Programming Languages" ],
+    "identity": {
+        "internalName": "[Replace]"
+    },
+    "overview": "overview.md",
+    "publisher": "microsoft",
```
Comment (BenJKuhn) [inline dev/VSIX/extension.manifest.json:8]: Capitalize?

File: dev/VSIX/overview.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The Microsoft Windows App SDK is a new set of developer components and tools that represents the next evolution in the Windows app development platform. It provides a unified set of APIs and tools that can be used in a consistent way by any app on a broad set of target Windows 10 OS versions.
+
+This is an experimental version of the Microsoft Windows App SDK. It contains features under development that may appear in future releases.
```
Comment (BenJKuhn) [inline dev/VSIX/overview.md:3]: Is this the same md that will be included in the release version? If so, please update this language.

File: dev/VSIX/extension.manifest.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vsix-publish",
+    "categories": [ "Build", "Coding", "Programming Languages" ],
+    "identity": {
+        "internalName": "[Replace]"
```
Comment (DefaultRyan) [inline dev/VSIX/extension.manifest.json:5]: Remember to update this line before finishing.

File: dev/VSIX/extension.manifest.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vsix-publish",
+    "categories": [ "Build", "Coding", "Programming Languages" ],
```
Comment (DefaultRyan) [inline dev/VSIX/extension.manifest.json:3]: The existing marketplace entries currently have the categories set to 'Templates" and "Other'. Not sure what "Other" refers to but "Templates" should be here, and I'm not sure about "Build", "Coding", and "Programming Languages".

**Review by DefaultRyan — CHANGES_REQUESTED** [review]

File: dev/VSIX/extension.manifest.json (hunk 3 above)
Comment (kythant) [reply dev/VSIX/extension.manifest.json:5]: This was going to be updated by the pipeline. I was planning on using this manifest json for all the VSIXs that we have.

File: dev/VSIX/overview.md (hunk 2 above)
Comment (kythant) [reply dev/VSIX/overview.md]: I've created a stable and preview overview. I've basically copied and pasted over the stable overview from the 0.8 so please take a look.

**Review by BenJKuhn — APPROVED** [review]

File: dev/VSIX/overview-preview.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+The Microsoft Windows App SDK is a new set of developer components and tools that represents the
```
Comment (DefaultRyan) [inline dev/VSIX/overview-preview.md:1]: overview-experimental.md would be a more appropriate filename.

**Review by DefaultRyan — APPROVED** [review]

**Pull request merged**